doctrine/multiple_entity_managers.rst

```diff
@@ -50,12 +50,21 @@ The following configuration code shows how you can configure two entity managers
                     default:
                         connection: default
                         mappings:
-                            AppBundle:  ~
-                            AcmeStoreBundle: ~
+                            App:
+                                is_bundle: false
+                                type: annotation
+                                dir: '%kernel.project_dir%/src/Entity'
+                                prefix: 'App\Entity'
+                                alias: App
                     customer:
                         connection: customer
                         mappings:
-                            AcmeCustomerBundle: ~
+                            AppCustomer:
+                                is_bundle: false
+                                type: annotation
+                                dir: '%kernel.project_dir%/src/CustomerEntity'
+                                prefix: 'App\Customer\Entity'
```

Comment:
I wonder if it's part of the entity namespace shouldn't be `dir: '%kernel.project_dir%/src/Customer/Entity'`? or vice-versa `prefix: 'App\CustomerEntity'`?

Reply:
I rather think that it should be `%kernel.project_dir%/src/Entity/Customer/`.

In full:

```yaml
AppCustomer:
    is_bundle: false
    type: annotation
    dir: '%kernel.project_dir%/src/Entity/Customer/'
    prefix: 'App\Entity\Customer'
```